src/Installers/Windows/SharedFrameworkLib/SharedFrameworkLib.wixproj

```diff
@@ -40,4 +40,23 @@
   <PropertyGroup>
     <PackageFileName>$(InternalInstallerBaseName)-$(PackageVersion)-win-$(Platform)$(TargetExt)</PackageFileName>
   </PropertyGroup>
+
+  <!-- Overwrite the wix package drop creation target to create a lit
+       package instead. -->
+  <Target Name="CreateWixPackageDrop" AfterTargets="Build">
+    <CreateLitCommandPackageDrop
+      LitCommandWorkingDir="$(WixCommandObjDir)"
+      WixExtensions="@(WixExtension)"
+      Bf="true"
+      Out="$(InstallersOutputPath)$(OutputName).wixlib"
+      WixSrcFiles="@(CompileObjOutput);@(WixObject);@(WixLibProjects);@(WixLibrary)">
+      <Output TaskParameter="LitCommandPackageNameOutput" PropertyName="_LitCommandPackageNameOutput" />
+    </CreateLitCommandPackageDrop>
```

Comment:
```suggestion
```

Reply:
Ignore me on this. Missed the `<Output />` element

Reply:
Doesn't this create invalid xml given the Output element above?

Reply:
Yup, that's why I said "ignore me" and resolved this point

Reply:
> Yup, that's why I said "ignore me" and resolved this point

I think we were clicking the buttons and typing at the same time.